repositories/postgres/postgres.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	defaultMaxPoolSize  = 1//3
+	defaultMaxPoolSize  = 5//3
```

Comment:
I made it 3 in my PR. Probably, let's try  from 3 for not having merge conflicts here?